backend/routes/doctorOrders.js

```diff
@@ -45,7 +45,7 @@ router.patch("/fhir/rems/pickedUp/:id", async (req, res, next) => {
 });
 
 
-router.post("/fhir/rems", (req, res, next) => {
+router.post("/$process-message", (req, res, next) => {
```

Comment:
change this to be ([\$])process-message. See https://github.com/expressjs/express/issues/2987 for more context